src/helpers.c

```diff
@@ -324,3 +324,13 @@ bool file_is_debuginfo(const char *path)
 	free(src);
 	return ret;
 }
+
+bool comp_ghosted(struct file *file1, struct file *file2)
+{
+	return (file1->is_ghosted && file2->is_ghosted);
+}
+
+bool comp_deleted(struct file *file1, struct file *file2)
```

Comment:
Also should move.

Maybe rename to both_deleted, I would expect from the name that the code would be
return (file1->is_deleted) == (file2->is_deleted);